Assets/Mirror/Runtime/SyncDictionary.cs

```diff
@@ -0,0 +1,331 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Mirror
+{
+    public class SyncDictionaryIntString : SyncDictionary<int, string>
```

Comment:
we will need to get the weaver to generate all these,  I don't think we need any hardcoded syncdict